starter-code/src/QueueDataStructure.js

```diff
@@ -1,2 +1,34 @@
-function QueueDataStructure () {
+function QueueDataStructure() {
+  this.queueControl = [];
+  this.MAX_SIZE = 8;
 }
+
+QueueDataStructure.prototype.isEmpty = function() {
+  if (this.queueControl.length === 0) {
```

Comment:
Just for your information, you can get rid of the if returning:
```
return this.queueControl.length === 0;
```
That returns a boolean, true when both values are equal, false if not. That said, your solution is good.